instruments/qubitekk/cc1.py

```diff
@@ -116,18 +116,23 @@ def count(self):
             :rtype: `int`
             """
             count = self._cc1.query("COUN:{0}?".format(self._chan))
-            # FIXME: Does this property actually work? The try block seems
-            # wrong.
+            tries = 5
             try:
                 count = int(count)
-            except ValueError:  # pragma: no cover
+            except ValueError:
                 count = None
-                while count is None:
+                while count is None and tries > 0:
```

Comment:
Introducing tries such that if count is forever `None`, it will end at some point. Five tries seem common across ik for such loops, but could make it a parameter.